src/Module.Server/ModuleData/native_parameters.xml

```diff
@@ -14,5 +14,8 @@
     <native_parameter id="running_backward_speed_multiplier" value="0.65" />  <!-- Native = 0.5 -->
     <native_parameter id="running_sides_speed_multiplier" value="0.775" />   <!-- Native = 0.65-->
     <native_parameter id="ladder_climb_up_speed_multiplier"  value="0.88" /> <!-- Native = 0.68-->
+    <native_parameter id="humanoid_physics_capsule_radius_multiplier_for_friend" value="1.3" /> <!--Native = 0.95-->

```

Comment:
You can do 1.5 here as the capsule are tight enough for us to have similar capsule for friend or enemy

Reply:
Ok :)